tests/test_fields.py

```diff
@@ -1,4 +1,92 @@
 from pykorm import fields
+from pykorm.models import Nested
+
+
+def test_nested_to_dict():
+    class TestObject(Nested):
+        foo = fields.DataField('foo')
+        bar = fields.DataField('bar')
+        foobar = fields.DataField('foobar', default='foobar')
+        foobarfoo = fields.DataField('foobarfoo', allow_empty=True)
+        foobarfoobar = fields.DataField('foobarfoobar')
+        foobarfoobarfoobar = fields.DataField(['foobarfoobarfoobar', 'foobarfoobarfoobar'])
+
+    ed = fields.DictNestedField(
+        TestObject, path='obj'
+    )
+
+    eds = fields.ListField(TestObject, path='objs')
+
+    wanted1 = {
+        'obj': {
+            'foo': 'foo',
+            'bar': 'bar',
+            'foobar': 'foobar',
+            'foobarfoo': None,
+            'foobarfoobarfoobar': {
+                'foobarfoobarfoobar': 'foobarfoobarfoobar'
+            }
+        }
+    }
+    wanted2 = {
+        'objs': [
+            {
+                'foo': 'foo1',
+                'bar': 'bar1',
+                'foobar': 'foobar',
+                'foobarfoo': None
+            },
+            {
+                'foo': 'foo2',
+                'bar': 'bar2',
+                'foobar': 'foobar',
+                'foobarfoo': None
+            },
+        ]
+    }
+    ed_dict = ed.to_dict(TestObject(foo='foo', bar='bar', foobarfoobarfoobar='foobarfoobarfoobar'))
+    assert ed_dict == wanted1
+    assert 'foobarfoobar' not in wanted1['obj']
+    eds_dict = eds.to_dict([TestObject(foo='foo1', bar='bar1'), TestObject(foo='foo2', bar='bar2')])
+    assert eds_dict == wanted2
+    for item in eds_dict['objs']:
+        assert 'foobarfoobar' not in item
+
+
+def test_nested_get_data():
+    class Two(Nested):
+        foo = fields.DataField('foo')
+        bar = fields.DataField('bar')
+        foobar = fields.DataField('foobar', default='foobar')
+
+    class One(Nested):
+        foo = fields.DataField('foo')
+        bar = fields.DataField('bar')
+        two = fields.DictNestedField(Two, path='two')
```

Comment:
could be differentiate the two classes and their arguments ? It's a bit confusing when comparing the two :D

Reply:
I changed the name of the class, see if it is more intuitive.